pkg/types/types.go

```diff
@@ -69,6 +67,17 @@ type RuntimeConfig struct {
 	CNIDeviceInfoFile string          `json:"CNIDeviceInfoFile,omitempty"`
 }
 
+// LoggingOptions specifies the configuration of the log
+type LoggingOptions struct {
+	LogFile     string `json:"logFile"`
+	LogLevel    string `json:"logLevel"`
+	LogToStderr bool   `json:"logToStderr,omitempty"`
+	MaxAge      int    `json:"maxAge,omitempty"`
+	MaxSize     int    `json:"maxSize,omitempty"`
+	MaxBackups  int    `json:"maxBackups,omitempty"`
+	Compress    bool   `json:"compress,omitempty"`
+}
+
```

Comment:
I would like to keep current logParameters in NetConf and add logging specific options are in LoggingOptions (logOptions) because this changes incompatible to previous version.

So setLogStderr()/setLogLevel() should be reverted as SetLogStderr()/SetLogLevel()/SetLogFile()

Reply:
Sorry, I didn't consider the compatibility.., Now I'm reverted it